app/src/main/java/de/danoeh/antennapod/ui/screen/queue/QueueFragment.java

```diff
@@ -596,6 +613,66 @@ protected void onSelectionChanged() {
         }
     }
 
+    /**
+     * This method checks if the selected items are allowed to be moved to the top, bottom or both of the Queue.
+     * @param queue The FeedItems currently in the Queue.
+     * @param selectedItems The FeedItems for which the check is performed.
+     * @return A boolean array where:
+     *           [0] is true if moving to the top is allowed, false otherwise.
+     *           [1] is true if moving to the bottom is allowed, false otherwise.
+     * */
+    public static boolean[] canMove(List<FeedItem> queue, List<FeedItem> selectedItems) {
```

Comment:
I think a `Pair<Boolean>` would be a bit more descriptive than a boolean array. When just looking at the method signature (without javadoc), this boolean array could have any length.